docs/Training-ML-Agents.md

```diff
@@ -197,6 +200,7 @@ are conducting, see:
 * [Training with PPO](Training-PPO.md)
 * [Using Recurrent Neural Networks](Feature-Memory.md)
 * [Training with Curriculum Learning](Training-Curriculum-Learning.md)
+* [Training with Generalization](Training-Generalization-Learning.md)
```

Comment:
Training with Generalization sounds weird to me. I would say Training with Environment Parameters Sampling